src/cosmosdb-continuous-backup/azext_cosmosdb_pitr/_help.py

```diff
@@ -0,0 +1,151 @@
+# coding=utf-8
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from knack.help_files import helps  # pylint: disable=unused-import
+
+
+helps['cosmosdb create'] = """
+type: command
+short-summary: Creates a new Azure Cosmos DB database account.
+parameters:
+  - name: --locations
+    short-summary: Add a location to the Cosmos DB database account
+    long-summary: |
+        Usage:          --locations KEY=VALUE [KEY=VALUE ...]
+        Required Keys:  regionName, failoverPriority
+        Optional Key:   isZoneRedundant
+        Default:        single region account in the location of the specified resource group.
+        Failover priority values are 0 for write regions and greater than 0 for read regions. A failover priority value must be unique and less than the total number of regions.
+        Multiple locations can be specified by using more than one `--locations` argument.
+  - name: --databases-to-restore
+    short-summary: Add a database and its collection names to restore
+    long-summary: |
+        Usage:          --databases-to-restore name=DatabaseName collections=collection1 [collection2 ...]
```

Comment:
`--databases-to-restore` could also specify multiple databases?

Reply:
yes. Multiple databases can be specified